datastream-server-restli/src/main/java/com/linkedin/datastream/server/dms/DatastreamResources.java

```diff
@@ -739,6 +740,51 @@ public ActionResult<Void> resumeSourcePartitions(@PathKeysParam PathKeys pathKey
     return new ActionResult<>(HttpStatus.S_200_OK);
   }
 
+  /**
+   * This endpoint handles the management of throughput violating topics. These topics' metrics and
+   * SLAs are reported separately as their throughput are not within brooklin's permissible bounds.
```

Comment:
This change is in the open source and Brooklin permissible bounds are very specific to Linkedin. 
Can we keep it generic? 

This endpoint gets the topic list to skip accounting in SLA and lag. How you generate the violation list is upto the use-case? 

For eg: Later we can use this for the topics with schema issues.

Reply:
Also, mention that the behavior is replace all, rather than incremental update.